src/Components/User/TODO.js

```diff
@@ -1 +1,122 @@
 // TODO
+import React from "react";
+import NavBar from "../Layout/NavBar";
+import CardTile from "../Workshop/CardTile";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Card from "react-bootstrap/Card";
+import Buttom from "react-bootstrap/Button";
+import ToggleButton from 'react-bootstrap/ToggleButton';
+import Button from "react-bootstrap/Button";
+import ProgressBar from "react-bootstrap/ProgressBar";
+import Container from "react-bootstrap/Container";
+
+
+
+
+class User extends React.Component {
+    constructor(props) {
+        super(props);
+
+        let first = {
+            Level_Descriptions: ["info part 1", "info part 2", "info part 3", 'part 4', 'part 5'],
+            Number_Of_Levels: 5,
+            Workshop_ID: "firebase",
+            Workshop_Name: "firebase",
+            Day: "Monday",
+            Date: "16",
+            Month: "March",
+            Year: "2020",
+            level_text: ['Setup the project', 'Researc the project', 'Learn the Project', 
+                        'Do the Project', 'Present the Project']
+            };
+
+          this.state = 
+          {
+              workshop_data: first,
+              progress: 0
```

Comment:
So this is where we need to use 2 database tables. One database table contains information on the workshop which is being used here. The other database table (check studentsAtWorkshop) contains information on students and progress. This class should contain information from the student table and then write that progress inforamtion to the studentsAtWorkshop table.